pkg/neorpc/filters.go

```diff
@@ -48,6 +51,43 @@ type (
 		Type   *mempoolevent.Type `json:"type,omitempty"`
 	}
 )
+type ValidFilter interface {
+	IsValid() error
+}
+
+// IsValid checks whether the filter is valid.
+func (f *BlockFilter) IsValid() error {
```

Comment:
Move it under `(f *BlockFilter) Copy()`, and move the rest of implementations correspondingly.